parent/pom.xml

```diff
@@ -5174,6 +5174,7 @@
             <Karaf-Info>Camel;${project.artifactId}=${project.version}</Karaf-Info>
             <_versionpolicy>${camel.osgi.import.default.version}</_versionpolicy>
             <_failok>${camel.osgi.failok}</_failok>
+            <Automatic-Module-Name>${camel.automatic.module.name}</Automatic-Module-Name>
```

Comment:
What happens here if other components do not have that option, is that value empty in the manifest file or ?

Reply:
Hi @davsclaus , yes if it's not defined there will be no value in the manifest (there won't even be a 'Automatic-Module-Name' key)